docker/entrypoint.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
```

Comment:
What is the purpose of this file?

Reply:
Bundled and used as the entrypoint to our docker image, see [Dockerfile](https://github.com/MarquezProject/marquez/blob/master/Dockerfile#L20)